cmd/client-fs.go

```diff
@@ -457,10 +457,6 @@ func deleteFile(deletePath string) error {
 	parentPath := strings.TrimSuffix(deletePath, slashSeperator)
 	parentPath = path.Dir(parentPath)
 
-	if parentPath != "." {
```

Comment:
Yeah, everything about `parentPath` can be removed.

Reply:
Done.